src/main/js/labeling/views/NavBarFactory.jsx

```diff
@@ -19,7 +19,11 @@ module.exports = function(WhoAmIStore) {
 				</div>
 				{this.props.piMode
 					? <h1>PI Information <small>for ICOS metadata</small></h1>
-					: <h1>ICOS Station Labeling <small>Steps 1, 2, 3</small></h1>}
+					: <div>
+							<h1>Labelled stations</h1>
```

Comment:
Should rather be "ICOS station labelling status", as it shows all the stations in the labelling pipeline, not only the labelled ones.